### ISSUE TITLE
It is extremely difficult to see yellow token and tile options when right-clicking

### ISSUE THREAD

**Issue by rtschaefer** [foundryvtt/foundryvtt#13581]
### User Experience

It used to be the case, that when you wanted to make a token or a tile invisible/visible, you could tell the current state easily because the icon was bright orange when active.  What seems to me as an unnecessary change, this is now a faint yellow color that is incredibly hard for me to see.  I have hunted for a setting that will allow me to change it back to the way it was, but I failed to find anything.  Please...please make a minor update that either reverts the color back to the way it was, or, offer a way for me to pick a color that I can actually see.  Thank you so much.
